framework/page_forms/spotify_home_form.py

```diff
@@ -0,0 +1,30 @@
+from forms.base_form import BaseForm
+from browser.py_quality_services import PyQualityServices
+from selenium.webdriver.common.by import By
+from resources.locator_constants import LocatorConstants
+
+
+class SpotifyHomeForm(BaseForm):
+    __form_name = "Popular artists"
+    __element = PyQualityServices.element_factory
```

Comment:
I think you already have a property for element factory in the BaseForm
_element_factory